src/components/Shared/StandardButton.vue

```diff
@@ -0,0 +1,27 @@
+<script setup>
+const props = defineProps({
+  type: {
+    type: String,
+    default: 'button'
+  },
+  text: String,
+  icon: Function,
+  activated: Boolean,
+  activatedText: String,
+  actiatedIcon: Function,
```

Comment:
typo. activated
and also can set v-if icon exist and other kind of logic

Reply:
activated props removed